sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -5121,6 +5121,11 @@ var setParentClass = function(child, parent) {
                 ret.operatorOrAssignBinary(tcuInterval.withNumbers(-Math.PI, Math.PI));
         }
 
+        if (ctx.format.hasInf() != tcuFloatFormat.YesNoMaybe.YES && (!yi.isFinite() || !xi.isFinite())) {
```

Comment:
see the native code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2458